include/torch-mlir/Conversion/TorchOnnxToTorch/Patterns.h

```diff
@@ -54,6 +54,21 @@ struct OpBinder {
     return success();
   }
 
+  ParseResult tensorOperands(SmallVector<Value> &dimList) {
+    int64_t numOperands = op->getNumOperands();
+    if (numOperands < 3)
```

Comment:
Why this failure? Seems like they're should not be a limit here

Reply:
I was thinking if numOperands < 3, the other tensorOperand methods should be used. But, will change so this method can be used regardless of numOperannds.

Reply:
The weird coupling isn't worth it and will cause hard to diagnose errors. Leave it as a caller choice too do one vs the other.